app/src/main/java/org/oppia/android/app/settings/profile/ProfileRenameActivityPresenter.kt

```diff
@@ -31,7 +31,7 @@ class ProfileRenameActivityPresenter @Inject constructor(
   }
 
   fun handleOnCreate() {
-    activity.title = activity.getString(R.string.profile_rename_title)
+    activity.title = activity.getString(R.string.profile_rename_activity_title)
```

Comment:
I think this can be removed. Can you try removing this and then checking if tests pass as well app works correctly: (a.) The toolbar on this activity shows correct title and (b.) The Talkback gives correct output.

Reply:
PTAL. That line was removed, and everything is working as expected. All the tests are passing + The title is being correctly displayed on the screen and the talkback is giving on the correct output.

![oppia_rt_conversation](https://user-images.githubusercontent.com/29144058/111729418-0a585d80-8895-11eb-8276-d72c0304a373.png)